### ISSUE TITLE
Logical grouping of statements/recipes

### ISSUE THREAD

**Issue by arc12** [jiscdev/xapi#131]
It occurs to me that, as the recipes stock grows it would be useful to show how sets of recipes are used together. This is likley to be useful particularly where the same recipe may figure in different situations. In some cases it is obvious (e.g. login/logout) but in others it may be less so (e.g. attending a meeting/interview following an early alert in an "intervention" process could use the same recipe as class attendance).

Proposal: we extend the documentation with a very light-weight page, e.g. called "application profiles" (or just revise the home README.md) which simply refers to the recipes applicable to a given application type.

General format could be e.g.:
**VLE Application Profile**
{URI for the VLE application subtype}
- name/link to recipe 
- ... other recipes ... 
_Any particular notes about the way context (etc) should be used to tie all the statements together_

**Comment by ds10** [comment]
The skype chat we had where this was discussed brought up some issues. This may just be a case of sign posting the repository better. Just some comments and notes that I remember from the call etc:

- @arc12 (if I remember correctly) had the idea that a recipe was a series of statements that were usful together. In this repository a recipe is a way of building a statement; this has followed on from the original Google Doc we inherited. If it is going to stay this way perhaps we should label them up as 'statement recipes'?
- We've got to a point now where recipes are going to have  cross over. Adam's example is a good one, the new intervention recipe could have a student attending a revision class and we already have an attendence statement recipe, so linking between them and explaining why they might be useful
- We have started to think about 'application profiles' I think. We have the VLE profiles already grouped and we have library recipes on the backburner. So perhaps this is the way to carry on thinking about it

**Comment by ds10** [comment]
I think I may be a bit confused here about what is needed.  

- We have statementsrecipes which are already grouped in some sense, 'VLE recipes', presence recipes, 'intervention recipes'. 
- We want to be able to group these some way, you've said 'application profile', but is it more like a analysists group? Like a group of statements that an analyist might want to see?
- The recipes are starting to have lots of additional information to them, which 'groupings' they might appear in, information on how the statement gets to the LRW (what should a TSV look like? Does a plugin create this statement?), What does the full statement look like?

I think the homepage might need a revising as well as a new page of groupings? That way analysts can go to the grouping page, but also from the list of recipes it is easy to see what kind of groups the statements belong to?

I've tried switching up the README in 0.6 branch to give an idea of what I mean. I think it would be useful for me to get a handle on the kinds of recipes you want to group together.

**Comment by arc12** [comment]
An associated aspect here is that some of the language is specific to a type of platform.
e.g. the formulation for login is actually NOT just for VLE login; the statememnt template applies more generally.

**Comment by ds10** [comment]
I think we left this and didn't close the ticket because while we revised the homepage, we weren't super happy with it but thought it was worth revisiting as the repoistory grew.

At present I think the recipes act as kind as application profiles? We have 'generic statement templates' that can be reused for a new profile. I think one thing we have struggled with is 'how do we group statements together'. A good example is the forum statement template for 1.1 .  The template kind of says 'if it is the first post do this, but if not then do that'. I have been revisiting old statements to refresh memory and think this would be an issue for a few statement templates, such as intervention.

The xAPI profiles spec has patterns, which describe statement templates groupings and how they should be ordered.

 https://github.com/adlnet/xapi-profiles/blob/master/xapi-profiles-structure.md#patterns

I think the approach would help some of the statement templates currently penciled in for 1.1 - particuarly forum usage?

@arc12 would this kind of grouping help
